firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionEvents.kt

```diff
@@ -74,4 +74,18 @@ internal object SessionEvents {
         ),
       applicationInfo = getApplicationInfo(firebaseApp)
     )
+
+  fun getApplicationInfo(firebaseApp: FirebaseApp): ApplicationInfo {
+    val packageName = firebaseApp.applicationContext.packageName
+    val packageInfo = firebaseApp.applicationContext.packageManager.getPackageInfo(packageName, 0)
```

Comment:
To avoid such a long include, can we import the package to make it look leaner?

Reply:
These aren't package imports, these are getters. Like `firebaseApp.getApplicationContext().getPackageManager()`. I can still make it look leaner by adding another val.